inception-imls-external/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/external/ExternalRecommender.java

```diff
@@ -280,9 +281,11 @@ private String getResponseBody(Response response) throws RecommendationException
     }
 
     @Override
-    public double evaluate(List<CAS> aCasses, DataSplitter aDataSplitter)
+    public EvaluationResult evaluate(List<CAS> aCasses, DataSplitter aDataSplitter)
     {
-        return 0;
+        EvaluationResult result = new EvaluationResult();
+        result.setDefaultScore(0.0);
```

Comment:
`return new EvaluationResult();` should be sufficient. See comment further down suggesting to add a flag indicating whether an evaluation was actually performed or not.